lisa/tools/git.py

```diff
@@ -201,6 +201,17 @@ def apply(
         )
         result.assert_exit_code(message=f"failed on applying patches. {result.stdout}")
 
+    def am(self, cwd: pathlib.PurePath, patch: pathlib.PurePath) -> None:
+        result = self.run(
+            f"am {patch}",
```

Comment:
does git am depend on b4 package?

Reply:
The output of b4 is a .mbx file. You can apply this patch to the repo using git am "filename.mbx"

As per description, git can be used to apply any email related patch, whereas git apply is for the .patch files

Reply:
I didn't see the place which uses b4 tool

Reply:
I will be raising that as a separate PR, added that in the description.
I have created a Modifier in Kernel Install transformer that uses b4 to apply patches

Reply:
I can add those changes to this PR, if you prefer having it in the same PR